stone/backends/tsd_helpers.py

```diff
@@ -101,8 +101,18 @@ def fmt_union(type_strings):
     """
     return '|'.join(type_strings) if len(type_strings) > 1 else type_strings[0]
 
-def fmt_func(name):
-    return fmt_camel(name)
+def fmt_func(name, version=1):
```

Comment:
Code in https://github.com/dropbox/stone/pull/99/files#diff-48ab1e69649131b8fb683a0484b22bafR129 needs to be updated to handle route docstring properly. See https://github.com/dropbox/stone/pull/98 for reference.

Unless this function will be used in circumstances where version is not available, it might be better to make `version` a required field to make sure people won't forget to pass the parameter.

Reply:
done. This was a good catch!  also added some versioned docstrings in tests to catch this.